src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -0,0 +1,110 @@
+    content: page_hero_banner_content
+%}
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+Firefox is adding support for Manifest Version 3 (MV3) extensions in early 2023. However, older versions of Firefox are only compatible with Manifest Version 2 (MV2) extensions. And, it is not possible to create a version of an extension that is MV2 and MV3 compatible.
+
+To help address the situation, AMO can distribute your MV2-compatible extension and sign an MV3-compatible version for [self distribution](/documentation/publish/self-distribution/). Some developers use this as a form of a beta channel, though it is not officially supported as such. 
+
+When deciding which version is most appropriate to distribute to your users through AMO, consider that:
+* When Firefox is updated in early 2023, it may take several weeks for most users to update from a version of the browser that only supports MV2. 
+* Older clients, such as Firefox ESR – the extended support release for enterprises (large companies and organizations) – will remain in use for several months (Firefox ESR 102.x is supported until September 2023 when the ESR release will move to a version supporting MV3).
+
+So, if your user base is large, or you have requirements to provide support for older Firefox versions for an extended period, we recommend that you consider using this alternate distribution for your MV3 version.
+
+If you choose to remain on MV2 and wait to transition to MV3 later, there are steps you can take in MV2 that [move your extension closer to what is necessary in MV3](https://blog.mozilla.org/addons/2022/10/31/begin-your-mv3-migration-by-implementing-new-features-today/). As usual, if your extension depends on features only available to recent Firefox versions, specify the compatible Firefox version in the [`strict_min_version`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) field in manifest.json. Older Firefox extensions will not receive an update, and a new user on an old Firefox version can install an older version of the extension using the “See all versions” link at the extension listing on AMO.
+
+{% endcapture %}
+{% include modules/column-w-toc.liquid,
+  id: "transfer-ownership"
+  content: content_with_toc
+%}
+
+<!-- END: Content with Table of Contents -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## How to sign an MV3 version
+
+To get a signed version of your MV3 extension:
+
+1. On AMO, when you upload the MV3 extension version, change “Where to host version”  to “on your own”.
+   ::: note
+   You need to change it back to “on this site” for MV2 versions, as the most recently used channel is remembered.
+   :::
+2. For API uploads, specify `channel=unlisted` for the self-distribution channel.
+
+See [Self distribution](/documentation/publish/submitting-an-add-on/#self-distribution), for more details on the process.
```

Comment:
```suggestion
See [self distribution](/documentation/publish/submitting-an-add-on/#self-distribution), for more details on the process.
```
?

Reply:
(we say "self distribution" earlier up the page, though for a different link)

Reply:
Because it matches the title of the page linked to.